addons/mail/__manifest__.py

```diff
@@ -259,6 +259,15 @@
             ('remove', 'web_tour/static/src/js/tour_manager.js'),
             ('remove', 'web_tour/static/src/js/tour_service.js'),
 
+            'web/static/src/views/fields/formatters.js',
+            'web/static/src/views/fields/file_handler.*',
+            'web/static/src/views/fields/many2many_tags/tags_list.*',
+            'mail/static/src/model/model_listener.js',
+            'mail/static/src/component_hooks/use_update.js',
+            'mail/static/src/new/*.js',
+            'mail/static/src/new/**/*.js',
+            ('remove', 'mail/static/src/new/public/**/*'),
+            ('remove', 'mail/static/src/new/views/**/*'),
             'mail/static/src/new/public/**/*',
```

Comment:
Keep separation of `web` requirements and `mail` ones. The long listing in `web` comes from bad bundle split in `web/`, so better keep ugliness together, so we know it's only `web/` that is annoying to maintain (and not `mail/` which is much nicer with folder = feature)